src/time.rs

```diff
@@ -152,12 +172,13 @@ impl Time {
             minute: ((second % 3600) / 60) as u8,
             second: (second % 60) as u8,
             microsecond,
+            tz_offset: None,
         })
     }
 
     /// Parse a time from bytes with a starting index, no check is performed for extract characters at
     /// the end of the string
-    pub(crate) fn parse_bytes_partial(bytes: &[u8], offset: usize) -> Result<(Self, usize), ParseError> {
+    pub(crate) fn parse_bytes_partial(bytes: &[u8], offset: usize) -> Result<Self, ParseError> {
```

Comment:
this needs to be renamed `parse_bytes` - it's no longer partial.

Reply:
But the existing `parse_bytes` method has a different signature. So if we combine them we will have:

`fn parse_bytes(bytes: &[u8], offset: usize)` with a required `offset`. Is that ok?
I was keeping `fn parse_bytes(bytes: &[u8])` to start parsing without index and this one with index.

Reply:
okay, I guess you can rename this to `parse_bytes_offset` - IIUC the point is it's no longer "partial" since it consumes the rest of the string.